### ISSUE TITLE
Structural model search tool for PKPD models

### ISSUE THREAD

**Issue by rikardn** [pharmpy/pharmpy#1743]
Develop the structural model search tool to handle PKPD models.

Procedure:

1. Input is a previously developed PK model
2. FIX all population parameters (this is PP&D. Other alternatives exists, but are out of scope)
3. For each category "direct response", "effect compartment" and "indirect response"
    a) Create baseline PD model and fit
    b) From baseline model create the remaining models in scope and fit all in parallel
4. Select best out of all tested models

Shortcut: start with only direct response models before adding the others.

**Comment by antoniaih** [comment]
We have implemented a workflow in structsearch for direct and effect compartment PD models.
Needs to be tested.

But we have encountered a problem: the input model is a PK model with PKPD data. It is not possible to fit the PK model to the PKPD data since the PD rows will be treated as PK observations. Thus the fit results for the PK input model will be wrong. Thus, we need to ignore all rows in the dataset where DVID=2 when fitting the PK model.
The best solution for this would be to add an IGNORE statement to the PK model before it is fitted and then removing the IGNORE statement before the PD fit. This is not possible at the moment. 

Current procedure is as follows: a copy of the PK model is created with a copy of the PKPD dataset. The dataset is then filtered (all rows with DVID=2 are removed). So we have a copy of the input model but with a filtered dataset. This is done BEFORE running the structsearch tool since it currently needs a `modelfit_results` object as input (there is a separate issue for this).
Filtering the dataset is not optimal since we do not want to change the dataset.
